### PR TITLE
Give mutations a more prominent spot in the docs, and add nested mutations

### PR DESCRIPTION
**Pull request by emilebosch**
Hey all,

Maybe mutations need to be more in the spotlight. Since its super common to do so, also i've added nested mutations which are super hawt 🙌 .

Mutations code /w help from @theorygeek.

### CODE REVIEW

File: guides/schema/mutations.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  # Create an input type for each item
+  TodoItemInputObjectType = GraphQL::InputObjectType.define do
+    name "TodoItem"
+    input_field :name, !types.String
+    input_field :starred, !types.Boolean
+  end
+
+  # Mutation takes an array of them:
+  input_field :todos, !types[!TodoItemInputObjectType]
+    end
```
Comment (Willianvdv) [inline guides/schema/mutations.md]: Unexpected end

Comment (emilebosch) [inline guides/schema/mutations.md]: I'll fix it in a minute and squash the commits 👍  thanks for catching bit of a hasty thing

File: guides/schema/mutations.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  name "AddTodoList"
+ 
+  # Create an input type for each item
+  TodoItemInputObjectType = GraphQL::InputObjectType.define do
+    name "TodoItem"
+    input_field :name, !types.String
+    input_field :starred, !types.Boolean
+  end
+
+  # Mutation takes an array of them:
```
Comment (Willianvdv) [inline guides/schema/mutations.md]: `them` should be `those`, right?

File: guides/schema/mutations.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  input_field :todos, !types[!TodoItemInputObjectType]
```
Comment (Willianvdv) [inline guides/schema/mutations.md]: Does the second `!` make a difference?

File: AUTHORS [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+Wenley Tong <wenley.tong@gmail.com>
+Wesley Ellis <tahnok@gmail.com>
+Zach Holman <zach@zachholman.com>
+Zach Kemp <zvkemp@gmail.com>
+Zoltan Olah <zol@percolatestudio.com>
+aviscasillas <aviscasillas@gmail.com>
+empunkt <mfa@gpredictive.de>
+exAspArk <exaspark@gmail.com>
+jordanielewski <jordanielewski@users.noreply.github.com>
+willian <willian@hackerone.com>
```
Comment (Willianvdv) [inline AUTHORS]: I'm listed multiple times with the same email address

File: guides/schema/mutations.md (hunk 3 above)
Comment (rmosolgo) [inline guides/schema/mutations.md]: Yes, the first one says "there will definitely be a list here (not `null`)" and the second one says "the members of the list will definitely be `TodoItemInputObjectType`s (there will be no `null`s in the list)"

File: AUTHORS [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+Mick Staugaard <mick@staugaard.com>
+Netto Farah <nettofarah@gmail.com>
+Nick Poorman <nickpoorman@gmail.com>
+Oguz Bilgic <fisyonet@gmail.com>
+Pol Miro <polmiro@gmail.com>
+Richard Lee <dlackty@gmail.com>
+Robin Murphy <robin@robinmurphy.co.uk>
+Robert Mosolgo <rdmosolgo@gmail.com>
+Christian Joudrey <cmallette@gmail.com>
+Garen Torikian <gjtorikian@gmail.com>
```
Comment (gjtorikian) [inline AUTHORS]: This list, while nice, is going to be impossible to keep updated.

I'm also really uncomfortable with the idea of a list that's just ripe for spammers to pluck from: "Hello I am working with an early stage dark horse pre-seed startup and we are using the GraphQLs to help revolutionize chicken farmers! Would you be open for a ten minute chat?"

Can we remove this?

Comment (emilebosch) [inline AUTHORS]: Sure, but its auto generated? Or is this maybe something from git-extras?

Comment (emilebosch) [inline AUTHORS]: Wait it is not. I see whats the problem.

**Comment by emilebosch** [comment]
Wait

**Pull request closed without merging**